api_docs/construct-narrow.md

```diff
@@ -51,8 +51,12 @@ important optimization when fetching messages in certain cases (e.g.
 when [adding the `read` flag to a user's personal
 messages](/api/update-message-flags-for-narrow)).
 
-**Changes**: In Zulip 9.0 (feature level 250), support was added for
-two filters related to channel messages: `channel` and `channels`. The
+**Changes**: In Zulip 9.0 (feature level 263), support was added for a
+filter related to followed topics: `is:followed`. This operator narrows to
+messages from topics that the user has [followed](/help/follow-a-topic).
```

Comment:
Change "messages from topics" => "message in topics"; that's more precise. This is also overly wordy; I think we could just say:

> **Changes**: In Zulip 9.0 (feature level 263), support was added for a
new `is:followed` filter, matching messages in topics that the current user is
[following](/help/follow-a-topic).